docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -2816,6 +2817,38 @@ Considerations for LOAD IF EXISTS and NO LOAD options of CREATE TABLE AS>>.
 +
 specifies a select query which is used to populate the created table. A select query can be any SQL select statement.
 
+* `_division-expr-list_`
++
+is a list of expressions that can only refer to primary key or STORE BY columns.
+
++
+The followings are allowed expressions in the DIVISON BY clause, they are monotonically increasing expressions:

```

Comment:
Grammar. Should be "The following are ..."

Reply:
Thanks Dave.
I've corrected. :)